site/docs/foundations/zindex.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Z-Index
+layout: DetailTechnical
+sidebar:
+  label: Z-Index
+data:
+  status: In progress
+---
+
+Z-index determines the stacking order of elements. By providing consistent z-indexes, the design system ensures a predictable hierarchy of visual elements when stacked on top of one another. 
+
+A component with a greater z-index, or stack order, will always appear in front of an element with a lower stack order.
+
+Z-index is often used in combination with [Shadow](shadow) to help distinguish each layer in the stack.
+
+## Z-Index tokens
+
+| Token                           | Value        |
+| ------------------------------- | ------------ |
+| `--salt-zIndex-flyover`         | 1,500        |
+| `--salt-zIndex-contextMenu`     | 1,450        |
+| `--salt-zIndex-dragObject`      | 1,420        |
+| `--salt-zIndex-notification`    | 1,400        |
+| `--salt-zIndex-modal`           | 1,300        |
+| `--salt-zIndex-drawer`          | 1,200        |
+| `--salt-zIndex-appHeader`       | 1,100        |
+| `--salt-zIndex-docked`          | 1,050        |
+| `--salt-zIndex-popout`          | 1,000        |
+| `--salt-zIndex-default`         | 1            |
+
+## Default
+`--salt-zIndex-default` represents the lowest layer in the stack. Its intended for any object that sits within the application layout, not over it. 
+
+## Popout
+`--salt-zIndex-popout`is used for any object that appears over the app layout in response to a user interaction and remains rooted to the element that triggered it, such as a [Tooltip](https://storybook.saltdesignsystem.com/?path=/docs/documentation-core-tooltip--page). 
+
+## Docked
```

Comment:
I'm not really sure what this would be used for

Reply:
Yes, I have no idea what Docked is used for! Do we have anyway of finding where its used in code? If we don't have a use for it we can remove it from the foundation guidance and as a token.